igvm/vm.py

```diff
@@ -713,3 +714,75 @@ def unblock_autostart(self):
 
     def copy_postboot_script(self, script):
         self.put('/buildvm-postboot', script, '0755')
+
+    def aws_disk_set(self, size: int) -> None:
+        """AWS disk set
+
+        Resize a disk in AWS.
+
+        :param: size: New disk_size
+
+        :raises: VMError: Generic exception for VM errors of all kinds
+        """
+
+        ec2 = boto3.resource('ec2')
+
+        response = ec2.Instance(self.dataset_obj['aws_instance_id'])
+        for vol in response.volumes.all():
+            volume_id = vol.id
+            break
+
+        ec2 = boto3.client('ec2')
+        ec2.modify_volume(VolumeId=volume_id, Size=int(size))
+
+        time.sleep(5)
+
+        with settings(
+            host_string=self.dataset_obj['hostname'],
+            warn_only=True,
+        ):
+            disk = sudo('lsblk -r | grep disk | cut -d " " -f1')
```

Comment:
Please never use sudo from Fabric, we have `vm.run()` which handles that.

Reply:
I changed all calls to use `vm.run()`